v4/export/config.go

```diff
@@ -246,6 +250,7 @@ func (conf *Config) DefineFlags(flags *pflag.FlagSet) {
 	flags.Bool(flagTransactionalConsistency, true, "Only support transactional consistency")
 	_ = flags.MarkHidden(flagTransactionalConsistency)
 	flags.StringP(flagCompress, "c", "", "Compress output file type, support 'gzip', 'no-compression' now")
+	flags.String(flagWriteSpeedLimit, "", "Dump phase network speed limit. No setting no limit.")
```

Comment:
```suggestion
	flags.String(flagWriteSpeedLimit, "", "Dump phase network speed limit. No setting no limit.")
	_ = flags.MarkHidden(flagWriteSpeedLimit)
```

Reply:
Why is that?

Reply:
resolved.

Reply:
I can't understand the unit and exact effect by reading this usage.

Reply:
please address this comment and let the one who has commented resolves it.

Reply:
> I can't understand the unit and exact effect by reading this usage.

Need to specify the `unit` used in the `usage` parameter in `flag`?

Reply:
explained in https://github.com/pingcap/dumpling/pull/275#issuecomment-882012503